pkgs/development/python-modules/cirq-rigetti/default.nix

```
@@ -1,46 +1,41 @@
 {
+  python,
+  fetchPypi,
   buildPythonPackage,
-  cirq-core,
   setuptools,
-  pyquil,
-  qcs-sdk-python,
-  pytestCheckHook,
+  numpy_1,
+  cirq-core,
 }:
 
+# Cirq-Rigetti is deprecated and limited to python 3.12 and Numpy 1
+# per https://github.com/quantumlib/Cirq/releases/tag/v1.5.0
+# For more details or to provide feedback see https://github.com/quantumlib/Cirq/issues/7058
+
 buildPythonPackage rec {
   pname = "cirq-rigetti";
-  pyproject = true;
-  inherit (cirq-core) version src;
-
-  sourceRoot = "${src.name}/${pname}";
+  version = "1.5.0";
+  format = "wheel";
 
-  pythonRelaxDeps = [ "pyquil" ];
+  disabled = (python.pythonVersion != "3.12");
```

```suggestion
  disabled = pythonAtLeast "3.13";
```
but is this really needed?

---

I couldn't think of `pythonAtLeast` to save my life. Thank you.

I'm basing this restriction on their release notes.

---

> I'm basing this restriction on their release notes.

If it builds anyway (and the tests pass), there's no need to make the restriction.